test/helper.rb

```diff
@@ -3,9 +3,7 @@
 require 'webmock/minitest'
 require 'pry'
 
-require 'ddtrace/encoding'
-require 'ddtrace/tracer'
-require 'ddtrace/span'
+require 'ddtrace'
```

Comment:
Turns out `test/default_test.rb`'s `require 'ddtrace'` was THE require statement that all other minitest tests in the `test:main` suite depended on.

I've moved it here, and replaced all statements with `require 'ddtrace'`, as all remaining minitest tests depend on the whole tracer to work.

Reply:
The few remaining Rails tests did not like `ddtrace` being eagerly required, so I've changed the controlled demolition strategy here.